content/docs/getting-started/accounts.md

```diff
@@ -29,9 +29,16 @@ Follow [these instructions to log in on the command line (CLI) and web UI (dashb
 
 <!--<img src="/img/cloud-gov-idp-button-cropped.png" alt="cloud.gov provider button to login page is highlighted in blue" width="373" height="187" />-->
 
-Your cloud.gov account requires multi-factor authentication. To log into the system, you need two "factors" -- something you know (your password) and something you have on your person (your smartphone).
+Your cloud.gov account requires multi-factor authentication: something you know (your password) and something you have on your person (your smartphone). In order to perform multi-factor authentication, you need an authentication application that generates time-based one-time passcodes, such as [Google Authenticator](https://support.google.com/accounts/answer/1066447?hl=en) or [Authy](https://www.authy.com/app/mobile). When you log into cloud.gov for the first time, follow the instructions to set this up.
```

Comment:
We had this explanation in our documentation before. The above defines MFA as what it means in both contexts. Though it might make sense to consider leveraging the same words we use on the login screens from 18F/Shibboleth-IdP3-TOTP-Auth#15. e.g. `MFA app` and `token code`.